webapp/commonComponents/expression/expressionVariables.js

```diff
@@ -8,37 +8,50 @@ import { sqlTypes } from '@common/surveyRdb/sqlTypes'
 import * as Expression from '@core/expressionParser/expression'
 
 // TODO: match all nodeDefTypes and throw an error if unknown:
-const toSqlType = nodeDef =>
-  NodeDef.isInteger(nodeDef) ? sqlTypes.integer : NodeDef.isDecimal(nodeDef) ? sqlTypes.decimal : sqlTypes.varchar
+const toSqlType = (nodeDef) => {
+  if (NodeDef.isInteger(nodeDef)) {
+    return sqlTypes.integer
+  }
+  if (NodeDef.isDecimal(nodeDef)) {
+    return sqlTypes.decimal
+  }
+  return sqlTypes.varchar
+}
+
+// Returns the name of the given nodeDef and appends "THIS" if it's equal to the current one
+const getVariableLabel = (nodeDef, nodeDefCurrent, i18n) =>
+  NodeDef.getName(nodeDef) +
+  (nodeDefCurrent && NodeDef.isEqual(nodeDef)(nodeDefCurrent) ? ` (${i18n.t('expressionEditor.this')})` : '')
```

Comment:
don't append THIS

Reply:
it has been asked by Lauri... he wanted to show the variable related to the current node definition with something like `<THIS>`, what should we write instead?

Reply:
I didn't see where Lauri asks to read 'This'.
Do not add anything

Reply:
ok